### PR TITLE
Updating cmdline to take the two arguments, per help

### PR DESCRIPTION
**Pull request by rshade**
Updated the `apikey save` to take the two args after, and print help if errors

### CODE REVIEW

**Comment by rshade** [comment]
@alejandrojnm would you mind reviewing this?

File: cmd/apikey_save.go [hunk 1]
```diff
@@ -44,13 +46,13 @@ var apikeySaveCmd = &cobra.Command{
 		var name, apiKey string
 		var err error
 
-		// if arg is more than one, return an error
-		if len(args) > 0 {
-			utility.Info("This command not need arguments")
+		// if arg is more than two, return an error
+		if len(args) > 2 {
+			utility.Info("There are to many arguments for this command")
```
Comment (andyjeffries) [inline cmd/apikey_save.go]: too

**Comment by andyjeffries** [comment]
Minor typo, but other than that LGTM 👍  @haardikdharma10 can you merge and include in the next release please. Thanks @rshade

**Comment by haardikdharma10** [comment]
LGTM. Thanks @rshade!

**Pull request merged**